### PR TITLE
Add footy points to the front end

### PR DESCRIPTION
**Pull request by sbetti02**
Just added the the footy points to the front end in a few locations. Also added some simple tests where possible. The one notable exception is that I didn't add  a test to the alumni directory because I think it requires more upfront cost to get that set up.

### CODE REVIEW

File: client/src/components/Header.js
Comment (reshadbinharun) [inline client/src/components/Header.js:9]: Get outta here with that python bullsh*t

File: client/src/components/Header.js
Comment (reshadbinharun) [inline client/src/components/Header.js]: I'm thinking whether we merge the ambassador portal in first and have the FootyPoints badge live in the welcomeBadge component?  Otherwise we have 3 components in the first column of the header which I can get with. Your call!

Comment (reshadbinharun) [reply client/src/components/Header.js]: image
Let's add some styling and be deliberate with stacking behavior.
Let's add `style = {{ margin: 5px }}` or better yet let's always get these component to stack vertically. Use a `Grid.Row`for accessContext dropdown and the badge

**Review by reshadbinharun — CHANGES_REQUESTED** [review]
Very clean and thank you for breaking ground with f/e tests!
Let's add some ✨  styling ✨  before we merge. Also refer to comment for welcome badge and let me know what you'd like to do!

File: client/src/components/FootyPoints.js
Comment (reshadbinharun) [inline client/src/components/FootyPoints.js]: Why are we rending Label as an anchor tag?

File: client/src/components/Header.js
Comment (sbetti02) [reply client/src/components/Header.js:9]: Lol I think there's actually some minor performance improvements for using import over require, and I think it's expected to be more of the standard in JS, though it's a newer feature

File: client/src/components/Header.js
Comment (sbetti02) [reply client/src/components/Header.js]: Hm interesting, and yeah I'm indifferent on which to merge in first, sounds like it should get moved to the welcomeBadge component, but I think it should just be a one line thing to get shifted over? And the supporting test should still work, so doesn't seem like a big deal to me either way

Comment (sbetti02) [reply client/src/components/Header.js]: Also yeah I'll look into the styling, good call. I was doing it without that dropdown present so it wasn't quite so obvious that it was out of line

File: client/src/components/FootyPoints.js
Comment (sbetti02) [reply client/src/components/FootyPoints.js]: Bc my brain is small and I just copied + pasted the code from the example

File: client/src/components/Header.js
Comment (sbetti02) [reply client/src/components/Header.js:9]: Also fuck you I like python :)

Comment (reshadbinharun) [reply client/src/components/Header.js:9]: Nothing wrong with using `import` that's a welcome ES6 thing, but I mean the 2 line spaces. It's like you don't even try my dude

**Review by focitti — APPROVED** [review]

File: client/src/components/AlumniDirectory.js
Comment (reshadbinharun) [inline client/src/components/AlumniDirectory.js]: Can we have this be in line with the alumnus's name?

File: client/src/components/AlumniProfile.js
Comment (reshadbinharun) [inline client/src/components/AlumniProfile.js]: Same here, inline with Alumnus's name instead of in a different section?

File: client/src/components/WelcomeBadge.js
Comment (reshadbinharun) [inline client/src/components/WelcomeBadge.js]: The formatting here is off can we fix this?
image

So it has flag first and appropriate padding like so
image

**Review by reshadbinharun — CHANGES_REQUESTED** [review]
Love the refactors! This PR is looking great, small suggestions.
Can we also attach screenshots to PR for f/e changes?

**Comment by sbetti02** [comment]
@reshadbinharun think all the style stuff should be updated as you were saying. I found it to be more trouble than it was worth to fit the footypoints badge on the same line as the name due to it being difficult to line up the text height and the icon height, so I put it on the next line. Screenshots attached, take a look and lmk what you think
Screen Shot 2021-03-03 at 11 25 35 PM
Screen Shot 2021-03-03 at 11 26 02 PM
Screen Shot 2021-03-03 at 11 26 11 PM
Screen Shot 2021-03-03 at 11 26 23 PM
Screen Shot 2021-03-03 at 11 26 57 PM

**Review by reshadbinharun — APPROVED** [review]
Beautiful! Let's lift offfff

**Pull request merged**
